CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to Forgerock's Javascript SDK
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Forgerock and its packages, which are hosted in the [ForgeRock organization](https://github.com/forgerock) on GitHub.
+
+### Git Flow
+
+Fork the repo to your own github account. From there, you can add the ForgeRock repository as an upstream.
+
+`git remote add upstream git@github.com:ForgeRock/forgerock-javascript-sdk.git`
+
+## Here is a checklist of items to ensure your Pull Request is reviewed and ready to be merged:
+
+- [ ] Please make all pull requests against `develop` branch
+- [ ] Fill out the Pull Request template
```

Comment:
Do we have any specific needs/wants about what goes in the PR template?

Reply:
we have a PR template file

Reply:
https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/.github/pull_request_template.md